<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphCalcView</class>
 <widget class="QWidget" name="GraphCalcView">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>840</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>991</width>
    <height>840</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>991</width>
    <height>840</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Graph Calculator</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: #191A19;
	color: white;
	font-family: Sans-serif;
}
QLineEdit {
	border: 1px solid #606060;
	border-radius: 25px;
	padding-left: 10px;
}
QSpinBox {
	border: 1px solid #606060;
	border-radius: 19px;
	padding-left: 10px;
}
QPushButton {
	border-radius: 25px;
	background-color: #374045;
}
QFrame {
	border: 1px solid #606060;
	border-radius: 19px;
	padding: 10px;
}

QCheckBox {
	border: 1px solid #606060;
	border-radius: 5px;
	padding-left: 5px;
}

</string>
  </property>
  <widget class="QPushButton" name="homeBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>770</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #374045;
	border-radius:20px;
}

QPushButton::hover {
	background-color: #3A4750;
}

QPushButton::pressed {
	background-color: #303841;
}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>661</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	color: white;
}</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string>0</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="mainLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>23</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	border: 0px;
}</string>
   </property>
   <property name="text">
    <string>y =</string>
   </property>
  </widget>
  <widget class="QPushButton" name="equalBtn">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>40</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #E16428;
}

QPushButton::hover {
	background-color: #ff8000;
}

QPushButton::pressed {
	background-color: #b35900;
}</string>
   </property>
   <property name="text">
    <string>=</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="plot" native="true">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>871</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="clearBtn">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>770</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #374045;
	border-radius:20px;
}

QPushButton::hover {
	background-color: #3A4750;
}

QPushButton::pressed {
	background-color: #303841;
}</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </widget>
  <widget class="QFrame" name="axisXFrame">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>111</y>
     <width>432</width>
     <height>131</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>5</number>
   </property>
   <widget class="QLabel" name="intervalLabelX">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans-serif</family>
      <pointsize>14</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	border: 0;
}</string>
    </property>
    <property name="text">
     <string>Interval</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="xAxisLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans-serif</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	border: 0;
}</string>
    </property>
    <property name="text">
     <string>X Axis</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="autoXBtn">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans-serif</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>auto</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="xStartSpin">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <number>-1000000</number>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
    <property name="value">
     <number>-10</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="xEndSpin">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <number>-1000000</number>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="axisYFrame">
   <property name="geometry">
    <rect>
     <x>499</x>
     <y>111</y>
     <width>431</width>
     <height>129</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>5</number>
   </property>
   <widget class="QLabel" name="intervalLabelY">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans-serif</family>
      <pointsize>14</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	border: 0;
}</string>
    </property>
    <property name="text">
     <string>Interval</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="yAxisLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans-serif</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	border: 0;
}</string>
    </property>
    <property name="text">
     <string>Y Axis</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="autoYBtn">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans-serif</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>auto</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="yEndSpin">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <number>-1000000</number>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="yStartSpin">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <number>-1000000</number>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
    <property name="value">
     <number>-1</number>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">View/GraphCalcView/QCustomPlot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
